.github/workflows/tests.yml

```diff
@@ -38,3 +38,66 @@ jobs:
     - name: Run pytest
       run: |
         poetry run python -m pytest -v tests/python
+
+  vecu:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.9', '3.10']
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - uses: Gr1N/setup-poetry@v7
+    - uses: actions/cache@v3
+      with:
+        path: ~/.cache/pypoetry/virtualenvs
+        key: ${{ runner.os }}-poetry-${{ hashFiles('poetry.lock') }}
+
+    - name: Install Dependencies
+      run: |
+        poetry install
+
+    - name: Spawn vECU
+      run: |
+        poetry run gallia vecu "tcp-lines://127.0.0.1:20162" rng --seed 3 --mandatory_sessions "[1, 2, 3]" --mandatory_services "[DiagnosticSessionControl, EcuReset, ReadDataByIdentifier, WriteDataByIdentifier, RoutineControl, SecurityAccess, ReadMemoryByAddress, WriteMemoryByAddress, RequestDownload, RequestUpload, TesterPresent]" &
+
+    - name: Test scan-services
+      run: |
+        poetry run gallia scan-services --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --sessions 1 2 --check-session
+
+    - name: Test scan-sessions
+      run: |
+        poetry run gallia scan-sessions --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --depth 2
+
+    - name: Test scan-identifiers
+      run: |
+        poetry run gallia scan-identifiers --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --start 0 --end 100
```

Comment:
I'm thinking about, if we should do this for a bunch of SIDs

Reply:
yeah. We should especially specify `--sid`. Currently the default is used, which is questionable.